.github/workflows/daily_modin_precommit.yml

```diff
@@ -471,19 +471,13 @@ jobs:
           python-version: '3.9'
       - name: Upgrade setuptools and pip
         run: python -m pip install -U setuptools pip
+      - name: Install tox
+        run: python -m pip install tox
       - name: Install protoc
         shell: bash
         run: .github/scripts/install_protoc.sh
-      - name: Install Snowpark
-        run: python -m pip install ".[modin-development]"
-      - name: Install Sphinx
-        run: python -m pip install sphinx
```

Comment:
Don't we still need these?

Reply:
It looks like the tox environment (`testenv:docs` in tox.ini) takes care of these.

Reply:
tox will do it for us